uuid_test.go

```diff
@@ -251,6 +251,20 @@ func TestFromBytesOrNil(t *testing.T) {
 	}
 }
 
+func TestMarshalJSON(t *testing.T) {
```

Comment:
I'd also add a couple of negative tests, eg. something that can't be parsed correctly, in all tests.